examples/auth-rust-demo/contracts/auth-rust-demo/src/entry.rs

```diff
@@ -12,10 +12,9 @@ use core::result::Result;
 
 use crate::error::Error;
 
-use crate::utils::generate_sighash_all::generate_sighash_all;
 use ckb_auth_rs::{
-    ckb_auth::{ckb_auth, CkbAuthError, CkbEntryType},
-    AuthAlgorithmIdType, CkbAuthType, EntryCategoryType,
+    ckb_auth::{ckb_auth, CkbEntryType},
```

Comment:
suggest to change `pub use ckb_auth::ckb_auth` and move `CkbEntryType` to lib.rs, then user may use it directly:

```
use ckb_auth_rs::{ckb_auth, CkbEntryType, CkbAuthType, ... }
```

and no need to define dummy `CkbEntryType` type in the unit test:
https://github.com/nervosnetwork/ckb-auth/blob/main/examples/auth-rust-demo/tests/src/tests.rs#L38-L42